quickstart/function_calling_calculate.ipynb

```diff
@@ -0,0 +1,181 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Function Calling with Llama 3.1: Getting started"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we’ll demonstrate how to use function calling with the Llama 3.1 model (70B) using a custom system prompt. We’ll integrate a simple function called calculateTool, which the model will be able to call when appropriate. The function attributes will be defined, and we’ll use the special llama-agentic-system syntax from Meta to pass the function in the system prompt."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Install these required dependencies to run this notebook\n",
+    "!pip install openai\n",
+    "!pip install python-dotenv==1.0.0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import libraries\n",
+    "import json\n",
+    "import re\n",
+    "from openai import OpenAI\n",
+    "from dotenv import load_dotenv\n",
+    "import os\n",
+    "\n",
+    "# Constants and Configuration\n",
+    "load_dotenv()\n",
+    "API_KEY = os.getenv(\"SAMBANOVA_API_KEY\")\n",
```

Comment:
Please change these to match our usual naming for consistency ie
SAMBANOVA_API_KEY
SAMBANOVA_BASE_URL